frontend/app/src/components/dashboard/NftBalanceTable.vue

```diff
@@ -177,6 +177,8 @@ watch(loading, async (isLoading, wasLoading) => {
           :loading="isLoading"
           :options="options"
           :server-items-length="itemLength"
+          multi-sort
+          :must-sort="false"
```

Comment:
Why disable must-sort is there any specific reason?

Reply:
So if we dont put it, dont we can remove all sort.
Actually I'm thinking now, we can set the default value of `mustSort` in the `<data-table />` to false

Reply:
> So if we dont put it, dont we can remove all sort. Actually, I'm thinking now, we can set the default value of `mustSort` in the `<data-table />` to false

I understand what the option itself does. If you check vuetify's documentation, By default `must-sort` is set to `false`, so it should be pretty obvious that this was not by accident.

We decided to enforce sorting by default all across the application at some point. This has been a deliberate decision.

My question was, about what made this case special and why did you want to introduce the ability to disable sorting in this specific case. Just so that by default it falls back to the backend's default?

Please revert this change if your only reason is to use the backend's default, otherwise please let me know, and we can see if it is a valid reason.

One of the reasons we removed the ability to have a default (neutral) sorting was that this was confusing to people and it was not clear how the data were sorted.

We are not changing the default across the application. This is a terrible idea.

Reply:
Okay, I just realized that we intended to set the `mustSort` to true by default.

I just understand the pain point here if we use `mustSort=false`.
I revert it, and also remove this in the other Non Fungible Balances in `Asset & Balances`